lib/TvmClient.js

```diff
@@ -440,8 +440,19 @@ TvmClient.AwsS3Endpoint = 'aws/s3'
 TvmClient.AzureCosmosEndpoint = 'azure/cosmos'
 TvmClient.inMemoryCache = null
 
+/* The global var AIO_ENV_IS_STAGE is expected to be set during build time via WEBPACK config */
+/* eslint no-undef: 0 */
+const stageEnv = (typeof AIO_ENV_IS_STAGE === 'undefined') ? false : AIO_ENV_IS_STAGE
```

Comment:
let's use `aio-lib-env` instead of reintroducing a new env var. Here is an example on how to implement this: https://github.com/adobe/aio-lib-ims/pull/66

Reply:
an interesting point is that this would give us support for `AIO_CLI_ENV=stage aio app deploy` to deploy to stage s3 for free